weblab/experiments/views.py

```diff
@@ -235,9 +235,14 @@ def get(self, request, *args, **kwargs):
             visibility_where = visibility_where | Q(entity__entity__in=visible_entities)
 
         # If specific versions have been requested, show at most those
-        q_model_versions = self.versions_query('model', model_versions, q_models, visibility_where)
-        protocol_visibility_where = visibility_where & Q(entity__entity__is_fitting_spec=False)
-        q_protocol_versions = self.versions_query('protocol', protocol_versions, q_protocols, protocol_visibility_where)
+
+        q_model_versions = self.versions_query('model', model_versions, q_models.values('pk'), visibility_where)
+        if show_fits:  # Temporary hack
```

Comment:
The merge has gone wrong here: the `show_fits` variable no longer exists in master. So you can just use the `else` clause definition for `protocol_visibility_where`.